kaldb/src/test/java/com/slack/kaldb/chunk/ReadOnlyChunkImplTest.java

```diff
@@ -64,57 +61,4 @@ public void setUp() throws IOException {
   public void tearDown() {
     registry.close();
   }
-
```

Comment:
I think it's best to wait until the next diff to merge this class along with it's tests. For now we can remove it.

Reply:
Removed